src/private/MDMPerformerGroup.m

```diff
@@ -72,6 +72,19 @@ - (void)executeLog:(MDMTransactionLog *)log trace:(MDMTrace *)trace {
     if ([performer respondsToSelector:@selector(addPlan:)]) {
       [(id<MDMPlanPerforming>)performer addPlan:plan];
     }
+    
+    // MDMNamedPlanPerforming callbacks
+    if (log.name.length) {
+      if (log.isRemoval) {
+        if ([performer respondsToSelector:@selector(removePlanNamed:)]) {
+          [(id<MDMNamedPlanPerforming>)performer removePlanNamed:log.name];
+        }
+      } else {
+        if ([performer respondsToSelector:@selector(addPlan:named:)]) {
+          [(id<MDMNamedPlanPerforming>)performer addPlan:plan named:log.name];
+        }
+      }
```

Comment:
The pseudo-code for adding/removing named plans in the performer group should look roughly like so:

```
# Step 1
performerForName(name).removePlan(named: name)

# Step 2
performer = performerForPlan(plan)
performer.add(plan: plan, withName: name)
performerForName(name) == performer 
> true
```

Remove-then-add: Two things happen when a named plan is committed:

1. Remove any previously committed plan with the same name from the target's performers.
  Note: This may be on a different performer instance on the same target. In the example above perhaps a PhysicsPerformer is needed for the second transaction, but not for the first.
2. Provide the relevant performer with the new named plan.